lib/tether/tether.go

```diff
@@ -315,28 +314,16 @@ func (t *tether) handleSessionExit(session *SessionConfig) {
 	session.Outwriter.Close()
 	session.Errwriter.Close()
 
-	// Log a death record trimming records if need be
-	logs := session.Diagnostics.ExitLogs
-	logCount := len(logs)
-	if logCount >= MaxDeathRecords {
-		logs = logs[logCount-MaxDeathRecords+1:]
-	}
-
 	// Remove associated PID file
 	cmdname := path.Base(session.Cmd.Path)
 	_ = os.Remove(fmt.Sprintf("%s.pid", path.Join(PIDFileDir(), cmdname)))
 
-	session.Diagnostics.ExitLogs = append(logs, executor.ExitLog{
```

Comment:
It wasn't being populated and after discussion w/ @hickeng we agreed to remove and use a single attribute